config/settings.yml

```diff
@@ -26,3 +26,6 @@ plugins:
   ldap_password:
     client: false
     default: ''
+  ldap_filter:
+    client: false
+    default: '(&(sAMAccountName=%{username})(memberof=CN=group-example,DC=example,DC=com))'
```

Comment:
Either `ldap_uid` or `ldap_filter` are required, right? Isn't having a default `ldap_filter` going to make `ldap_uid` irrelevant? We can add this but maybe it should not have a default value ie default to nil.